src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colours.values().length);
```

Comment:
move Random to the class field, like:
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/cd2ac733-16de-41f9-a612-b0164f4a7f81)